routes/admin.js

```diff
@@ -94,7 +94,10 @@ router.get('/users', function (req, res, next) {
     //   filter: URI encoded JSON, e.g. {"name":"michael"}
     //   order_by: e.g. name%20DESC, or customId%20ASC
     //   no_cache: 0 or 1, set to 1 to really count the records, otherwise the result may be read from a cache
-    utils.getFromAsync(req, res, '/registrations/pools/wicked', 200, function (err, apiResponse) {
+    var filterFields  = ['id','name','email'];
```

Comment:
Please use `const` or `let` for new code; I am phasing out all usage of `var`. Thanks.